pyaedt/edb_core/padstack.py

```diff
@@ -12,11 +12,11 @@
     from System import Double, Array
     from System.Collections.Generic import List
 except ImportError:
-    warnings.warn('This module requires pythonnet.')
+    warnings.warn('This module requires Python.NET.')
```

Comment:
```suggestion
    warnings.warn('This module requires `pythonnet`.')
```
Same comment as above.